website/docs/r/storage_object_bucket.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "scaleway"
+page_title: "Scaleway: scaleway_storage_object_bucket"
+sidebar_current: "docs-scaleway-resource-storage-object-bucket"
+description: |-
+  Manages Scaleway object storage buckets.
+---
+
+# scaleway_storage_object_bucket
+
+Creates and manages Scaleway object storage buckets. For more information, see [the documentation](https://www.scaleway.com/en/docs/object-storage-feature/).
+
+## Example Usage
+
+```hcl
+resource "scaleway_storage_object_bucket" "some_bucket" {
+    name = "some-unique-name"
+    acl = "private"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - The name of the bucket.
+* `acl` - (Optional) The canned ACL you want to apply to the bucket.
+* `region` - (Optional) The [region](https://developers.scaleway.com/en/quickstart/#region-definition) you want to attach this resource to.
```

Comment:
Like [here](https://github.com/terraform-providers/terraform-provider-scaleway/pull/193/files#diff-5c95b28312e3fd1bb991cb7cd9561937R153): `The [region](https://developers.scaleway.com/en/quickstart/#region-definition) in which the server should be created.`